django/contrib/admin/static/admin/js/core.js

```diff
@@ -85,6 +85,18 @@ function findPosY(obj) {
         return (this.getSeconds() < 10) ? '0' + this.getSeconds() : this.getSeconds();
     };
 
+    Date.prototype.getAbbrevMonthName = function() {
+        return typeof window.CalendarNamespace === "undefined"
+            ? this.getTwoDigitMonth()
+            : this.getFullMonthName().substring(0, 3);
```

Comment:
I don't think this will work in all languages. Probably there should be two sets of translations.